### PR TITLE
Add rebuild toc button to admin panel

### PR DESCRIPTION
**Pull request by ArthurZaharov**

### CODE REVIEW

File: assets/javascripts/discourse/models/topic.js.es6 [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default Discourse.Topic.reopen({
     var self = this;
 
     return Discourse.ajax(
-      '/table_contents/' + this.get('id'),
+      '/table_contents/',
```
Comment (VladimirMikhailov) [inline assets/javascripts/discourse/models/topic.js.es6]: `/table_contents'

**Comment by VladimirMikhailov** [comment]
Could you remove "Build A TOC" functionality from topic admin menu?

**Comment by ArthurZaharov** [comment]
I think we need to confirm this functionality before.

**Comment by VladimirMikhailov** [comment]
:ok:

**Pull request merged**